cmd/network/packet-capture_test.go

```diff
@@ -31,3 +56,902 @@ func TestPacketCaptureCmdComplete(t *testing.T) {
 		})
 	}
 }
+
+func TestNewPacketCaptureOptions(t *testing.T) {
+	streams := genericclioptions.IOStreams{}
+	lazyClient := &k8s.LazyClient{}
+
+	ops := newPacketCaptureOptions(streams, lazyClient)
+
+	assert.NotNil(t, ops)
+	assert.Equal(t, streams, ops.IOStreams)
+	assert.Equal(t, lazyClient, ops.kubeCli)
+}
+
+func TestComplete(t *testing.T) {
+	tests := []struct {
+		name    string
+		reason  string
+		wantErr bool
+	}{
+		{
+			name:    "without reason",
+			reason:  "",
+			wantErr: false,
+		},
+		{
+			name:    "with reason",
+			reason:  "OHSS-1234",
+			wantErr: false,
+		},
+	}
+
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			ops := &packetCaptureOptions{
+				reason:  tt.reason,
+				kubeCli: &k8s.LazyClient{},
+			}
+			err := ops.complete(nil, nil)
+			if tt.wantErr {
+				assert.Error(t, err)
+			} else {
+				assert.NoError(t, err)
+			}
+		})
+	}
+}
+
+func TestDesiredPacketCaptureDaemonSet(t *testing.T) {
+	ops := &packetCaptureOptions{
+		name:             "test-capture",
+		namespace:        "default",
+		nodeLabelKey:     "node-role.kubernetes.io/worker",
+		nodeLabelValue:   "",
+		duration:         60,
+		captureInterface: "eth0",
+	}
+
+	key := types.NamespacedName{Name: ops.name, Namespace: ops.namespace}
+	ds := desiredPacketCaptureDaemonSet(ops, key)
+
+	assert.Equal(t, ops.name, ds.Name)
+	assert.Equal(t, ops.namespace, ds.Namespace)
+	assert.Equal(t, map[string]string{"app": ops.name}, ds.Spec.Selector.MatchLabels)
+	assert.Equal(t, true, ds.Spec.Template.Spec.HostNetwork)
+	assert.Equal(t, 1, len(ds.Spec.Template.Spec.InitContainers))
+	assert.Equal(t, 1, len(ds.Spec.Template.Spec.Containers))
+}
+
+func TestDesiredPacketCapturePodExist(t *testing.T) {
+	ops := &packetCaptureOptions{
+		name:             "test-capture",
+		namespace:        "default",
+		nodeLabelKey:     "node-role.kubernetes.io/worker",
+		nodeLabelValue:   "",
+		duration:         60,
+		captureInterface: "eth0",
+	}
+
+	key := types.NamespacedName{Name: ops.name, Namespace: ops.namespace}
+	pod := desiredPacketCapturePod(ops, key)
+
+	assert.Equal(t, ops.name, pod.Name)
+	assert.Equal(t, ops.namespace, pod.Namespace)
+	assert.Equal(t, map[string]string{"app": ops.name}, pod.Labels)
+	assert.Equal(t, true, pod.Spec.HostNetwork)
+	assert.Equal(t, 1, len(pod.Spec.InitContainers))
+	assert.Equal(t, 1, len(pod.Spec.Containers))
+}
+
+func TestNewPacketCaptureOptionsOne(t *testing.T) {
+	streams := genericclioptions.IOStreams{}
+	lazyClient := &k8s.LazyClient{}
+
+	ops := newPacketCaptureOptions(streams, lazyClient)
+
+	assert.NotNil(t, ops)
+	assert.Equal(t, streams, ops.IOStreams)
+	assert.Equal(t, lazyClient, ops.kubeCli)
+}
+
+func TestDesiredPacketCaptureDaemonSetOne(t *testing.T) {
```

Comment:
How is this different from `TestDesiredPacketCaptureDaemonSet`?  Should they be grouped?